packages/reason-react/reason-react.0.12.0/opam

```diff
@@ -0,0 +1,56 @@
+opam-version: "2.0"
+synopsis: "Reason bindings for React.js"
+description: """
+ReasonReact helps you use Reason to build React components with deeply integrated, strong, static type safety.
+
+It is designed and built by people using Reason and React in large, mission critical production React codebases."""
+maintainer: [
+  "David Sancho <dsnxmoreno@gmail.com>"
+  "Antonio Monteiro <anmonteiro@gmail.com>"
+]
+authors: [
+  "Cheng Lou <chenglou92@gmail.com>" "Ricky Vetter <rickywvetter@gmail.com>"
+]
+license: "MIT"
+homepage: "https://reasonml.github.io/reason-react"
+doc: "https://reasonml.github.io/reason-react"
+bug-reports: "https://github.com/reasonml/reason-react/issues"
+depends: [
+  "dune" {>= "3.9"}
+  "ocaml" {>= "4.13.0"}
+  "melange" {>= "1.0.0"}
+  "reason-react-ppx" {= version}
+  "reason" {>= "3.6.0" & < "3.10.0"}
+  "ocaml-lsp-server" {with-test}
+  "opam-check-npm-deps" {= "1.0.0" & with-dev-setup}
+  "ocamlformat" {= "0.24.0" & with-dev-setup}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
```

Comment:
```suggestion
  ["dune" "build" "-j" "jobs" "@runtest"] {with-test}
]
```

Reply:
I removed the tests on CI in [`c6becbd` (#24422)](https://github.com/ocaml/opam-repository/pull/24422/commits/c6becbd5b112a37fb4120099b0f768191c79f47a) since it broke on execution, complaining that relies on reason-react which is a mutual-dependant dependency. Since I handle the CI and this "relation" on the repo I'm happy to remove the testing when installing to users.

Reply:
If you want, you can use `"reason-react" {with-test & =version & post}` to break the chain: https://opam.ocaml.org/doc/Manual.html#pkgvar-post

Reply:
I didn't know about `post`. Maybe all all packages from the same "dune-project" should have it? 

Nevermind, I should try but happy to not run tests on install

Reply:
If not running test on the CI is acceptable to you, we can definitely omit it.

What I have suggested is a hack: when `with-test` is present, separated from the main build step, it adds a step to test all packages in the downloaded source, which includes the ppx package as well.

But that is when I didn't know about the `post` condition mention in https://github.com/ocaml/opam-repository/pull/24422/files#r1325843840. That should be the preferred way.